File: demo_app/app/src/main/java/com/bandyer/demo_android_sdk/utils/networking/DemoAppUsers.java
/*
 * Copyright (C) 2018 Bandyer S.r.l. All Rights Reserved.
 * See LICENSE.txt for licensing information
 */

package com.bandyer.demo_android_sdk.utils.networking;

import java.util.List;

/**
 * @author kristiyan
 */

public class DemoAppUsers {

    public List<DemoAppUser> results;

    public class DemoAppUser {
        public Name name;

        public String email;

        public Picture picture;


        public class Name {
            public String first;
            public String last;
        }

        public class Picture {
            public String large;
        }
    }

}

File: app_utilities/src/main/java/com/bandyer/app_utilities/networking/DemoAppUsers.kt
/*
 * Copyright (C) 2020 Bandyer S.r.l. All Rights Reserved.
 * See LICENSE.txt for licensing information
 */
package com.bandyer.app_utilities.networking

/**
 * @author kristiyan
 */
class DemoAppUsers {
    @JvmField
    var results: List<DemoAppUser>? = null
    @JvmField
    var info: DemoUserInfo? = null

    inner class DemoAppUser {
        @JvmField
        var name: Name? = null
        @JvmField
        var email: String? = null
        @JvmField
        var picture: Picture? = null

        inner class Name {
            @JvmField
            var first: String? = null
            @JvmField
            var last: String? = null
        }

        inner class Picture {
            @JvmField
            var large: String? = null
        }
    }
}